src/ImageService/Controllers/ImageNewsController.cs

```diff
@@ -1,21 +1,31 @@
-﻿using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+﻿using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+using LT.DigitalOffice.ImageService.Models.Dto.Requests;
 using LT.DigitalOffice.ImageService.Models.Dto.Responses;
 using LT.DigitalOffice.Kernel.Responses;
 using Microsoft.AspNetCore.Mvc;
-using System;
 
 namespace LT.DigitalOffice.ImageService.Controllers
 {
-    [ApiController]
-    [Route("[controller]")]
-    public class ImageNewsController
+  [ApiController]
+  [Route("[controller]")]
+  public class ImageNewsController : ControllerBase
+  {
+    [HttpGet("get")]
+    public OperationResultResponse<ImageDataResponse> Get(
+      [FromServices] IGetImageNewsCommand command,
+      [FromQuery] Guid imageId)
     {
-        [HttpGet("get")]
-        public OperationResultResponse<ImageDataResponse> Get(
-            [FromServices] IGetImageNewsCommand command,
-            [FromQuery] Guid imageId)
-        {
-            return command.Execute(imageId);
-        }
+      return command.Execute(imageId);
     }
+
+    [HttpPost("create")]
+    public OperationResultResponse<List<Guid>> Create(
+      [FromServices] ICreateImageNewsCommand command,
+      [FromBody] CreateImageRequest request)
```

Comment:
why do you do like this? How i understand you create only one image but you return list of guids

Reply:
yea, you want to return image id and id of preview but how can i understand which id goes first? Maybe you should to create new entity. which contains two properties ImageId and PreviewId

Reply:
done